### PR TITLE
feat: initial working version

### PR DESCRIPTION
**Pull request by mzahor**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "name": "aspecto-sns-sqs-client",
+    "version": "1.0.0",
+    "main": "index.js",
+    "license": "MIT",
+    "scripts": {
+        "test": "jest"
+    },
+    "dependencies": {
+        "aws-sdk": "^2.642.0",
```
Comment (habmic) [inline package.json]: Shouldn't it be a peer dependency?

**Comment by habmic** [comment]
Just two notes:

1. Maybe we should allow to get an external logger? that way developer can use their own logger.
2. We relay on node env var to let the AWS SDK to perform the authentication, which is fine but we need to explain that in the docs

File: package.json (hunk 1 above)
Comment (mzahor) [reply package.json]: u r right, it should, will fix

**Comment by mzahor** [comment]
1. I thought about it, so I've added an event emitter exactly for this reason. So in order to add logs all you need to do is smth like this:
```ts
const sqsConsumer = SqsConsumer.create();
sqsConsumer.on(SqsConsumerEvents.processingError, (err) => logger.error(err))
```

**Comment by mzahor** [comment]
2. regarding docs - good point, will add.

**Pull request merged**